nixos/modules/services/home-automation/ebusd.nix

```diff
@@ -169,7 +169,7 @@ in
         LockPersonality = true;
         MemoryDenyWriteExecute = false;
         NoNewPrivileges = true;
-        PrivateDevices = usesDev;
+        PrivateDevices = !usesDev;
```

Comment:
What if the device is ens:DEVICE or enh:DEVICE? Does that usage already work with PrivateDevices = true?

Reply:
Oh, I missed that. I made some changes and it also works now with ens:DEVICE and enh:DEVICE.